go/felix/usagerep/usagerep.go

```diff
@@ -54,6 +50,19 @@ func PeriodicallyReportUsage(interval time.Duration, hostname, clusterGUID, clus
 	}
 }
 
+func calculateInitialDelay(numHosts int) time.Duration {
+	// To avoid thundering herd, inject some startup jitter.
+	if numHosts <= 0 {
+		// Intn requires a positive argument so clamp numHosts to be >= 1.
+		numHosts = 1
+	}
+	initialJitter := time.Duration(rand.Intn(numHosts*1000)) * time.Millisecond
```

Comment:
It's correct: `time.Duration` is a type, not a method.  `time.Duration(...)` is a cast, which is required by go's (rather minimal) promotion rules.  `time.Millisecond` is a `time.Duration` constant.